stable/sonatype-nexus/templates/deployment.yaml

```diff
@@ -120,12 +120,23 @@ spec:
         - name: nexus-backup
           image: {{ .Values.nexusBackup.imageName }}:{{ .Values.nexusBackup.imageTag }}
           imagePullPolicy: {{ .Values.nexusBackup.imagePullPolicy }}
+          resources:
```

Comment:
Let's make the resources configurable just pass the entire object from the values.